src/mixins/amalgamation-mixin.ts

```diff
@@ -221,9 +219,9 @@ export default class AmalgamationMixin extends Vue {
           role: AmlRoles.AMALGAMATING,
           identifier: tingBusiness.businessInfo.identifier,
           name: tingBusiness.businessInfo.legalName,
-          email: tingBusiness.authInfo.contacts[0].email,
+          email: tingBusiness.authInfo?.contacts[0].email,
```

Comment:
If authInfo is falsy then email is assigned undefined. Would it better if it was assigned null (ie, add `|| null`)?

Ditto line 224.